### ISSUE TITLE
chrome-extension protocol scheme support in Electron 5.0.x

### ISSUE THREAD

**Issue by sanoopjose** [electron/electron#19246]
CORS policy not allowing to access chrome extension resources using chrome-extension:// protocol policy in the latest version on Electron(5.0.x);

### Issue Details

* **Electron Version:**
  * 5.0.4
* **Operating System:**
  * 10.14.5 (18F132)
* **Last Known Working Electron version:**
  * 3.1.12

### Expected Behavior
Support chrome-extension:// protocol scheme on cross-origin resource access from chrome extension content script.

### Actual Behavior
Getting following error while 
```
Access to XMLHttpRequest at 'chrome-extension://<extension Id>/test.html' from origin 'https://calendar.google.com' has been blocked by CORS policy: Cross-origin requests are only supported for protocol schemes: http, data, chrome, https.
```

### To Reproduce
This issue is happening whenever you try to make a  CORS request from the chrome extension content script. I have tested this in the Chromium browser(73.0.3683.0) and was working fine there. After enabling Cisco Webex Scheduler(ID: ankblejcdieljecjfagjeoelaedaoaka), and navigate to Google calendar new event page and check the developer tools.

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.1.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
